src/hooks-worker-client.coffee

```diff
@@ -9,23 +9,19 @@ logger = require './logger'
 which = require './which'
 
 
-HOOK_TIMEOUT = 5000
-
-CONNECT_TIMEOUT = 1500
-CONNECT_RETRY = 500
-AFTER_CONNECT_WAIT = 100
-
-TERM_TIMEOUT = 5000
-TERM_RETRY = 500
-
-HANDLER_HOST = 'localhost'
-HANDLER_PORT = 61321
-HANDLER_MESSAGE_DELIMITER = "\n"
-
-
 class HooksWorkerClient
   constructor: (@runner) ->
-    @language = @runner.hooks?.configuration?.options?.language
+    options = @runner.hooks.configuration.options
```

Comment:
I'm so sorry for the delayed response. I'd use `@runnner.configuration`.

I'd be up for removing `hooks.configuration` to prevent data duplication and having a single source of truth in the future.